src/fu-udev-device.c

```diff
@@ -194,6 +194,8 @@ fu_udev_device_probe (FuDevice *device, GError **error)
 			tmp = g_udev_device_get_property (priv->udev_device, "ID_MODEL_FROM_DATABASE");
 		if (tmp == NULL)
 			tmp = g_udev_device_get_property (priv->udev_device, "ID_MODEL");
+		if (tmp == NULL)
+			tmp = g_udev_device_get_property (priv->udev_device, "ID_PCI_CLASS_FROM_DATABASE");
```

Comment:
I guess it's not ideal, but at this point we're kinda clutching at straws.

Reply:
I was thinking the same, but consider what we're doing for synaptics touchpad.  Then at least it's classed right.